pkg/userdata/token/token.go

```diff
@@ -57,3 +57,32 @@ func (t *Token) Expired() bool {
 
 	return t2-t1 < BootstrapTTL
 }
+
+func (t *Token) String() string {
+	return t.uuid.String()
+}
+
+// UnmarshalYAML implements the unmarshaller interface so we can
+// represent a uuidv4 token as a string
```

Comment:
nits:

- End with a period (This will read as a proper sentence in GoDoc).
- `s/uuidv4/UUID V1/`